### DISCUSSION TITLE
Fuzzy finder(fzf+vim) 使用全指南

### DISCUSSION THREAD

**Discussion by giscus** [keelii/fresh-blog#9] [Announcements]
# Fuzzy finder(fzf+vim) 使用全指南

芝兰生于深林，不以无人而不芳，君子修道立德，不为穷困而败节。

https://keelii.com/2018/08/13/fuzzy-finder-full-guide

**Comment by one-step-per-week** [comment]
Very good and detailed intro on fzf + vim!
